changelog.md

```diff
@@ -3,6 +3,26 @@
 A short month-by-month synopsis of change highlights. Most bugfixes won't make
 it in here, only the bigger features and interface changes.
 
+# Important changes between release 2019.04 and 2019.10
+
+* opm-common and the rest of OPM does not use libecl anymore and
+  supports reading and writing Eclipse files directly
+* improved Eclipse compatible restart support (including e.g. UDQ
```

Comment:
The UDQ and ACTIONX support is not really complete when it comes to restart.

Reply:
then please rephrase.

Reply:
I would just remove the full bullet point.